entries/e/epayslips.co.uk.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Epayslips": {
+    "domain": "epayslips.co.uk",
+    "tfa": [
+      "u2f",
+      "custom-software"
+    ],
+    "categories": [
+      "finance"
+    ],
+    "custom-software": [
+      "Authy",
+      "Duo"
```

Comment:
Their security page mentions Google Authenticator and Duo but nothing about Authy. It also sounds more like they support TOTP (RFC-6238) rather than the proprietary Duo and Authy authentication systems.
![Screenshot](https://github.com/2factorauth/twofactorauth/assets/3535780/3b53c783-8bdf-4685-a2b3-cef0adbe9f3c)

Reply:
I was just going off of the issue that is linked to this pull request but the documentation is probably better

Reply:
You need to remove the `custom-software` element then and replace `custom-software` inside `tfa` with `totp`.